packages/actions/src/utils/toChecksumAddress.js

```
@@ -0,0 +1,41 @@
+
+  // Hash the lowercase address using Keccak-256
+  let hash = keccak256(stringToHex(addressLower));
+
+
+// Iterate over each character of the address and the corresponding hash
+let checksumAddress = '0x';
+for (let i = 0; i < addressLower.length; i++) {
+  if(hash[i]){
+
+  // @ts-ignore
+    if (parseInt(hash[i], 16) >= 8) {
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)